drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/common/DecisionTablePopoverUtils.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.workbench.screens.guided.dtable.client.wizard.column.pages.common;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import com.google.gwt.dom.client.Element;
+import org.jboss.errai.common.client.dom.HTMLElement;
+
+@ApplicationScoped
+public class DecisionTablePopoverUtils {
+
+    private final List<Element> popoverElementRegistrations = new ArrayList<>();
+    private final List<HTMLElement> popoverHTMLElementRegistrations = new ArrayList<>();
+
+    public void setupPopover(final Element e,
+                             final String content,
+                             final boolean register) {
```

Comment:
Personally, I would add two methods instead of using the `register` parameter. Something like:
```java
setupPopover(final Element e, final String content)
// and
setupAndRegisterPopover(final Element e, final String content)
```
However, feel free to keep it as is ;-)